jetbrains-core/resources/META-INF/ext-terminal.xml

```diff
@@ -0,0 +1,15 @@
+<!-- Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<idea-plugin>
+    <actions>
+        <action id="aws.toolkit.startTerminal" class="software.aws.toolkits.jetbrains.core.terminal.OpenAwsLocalTerminal"/>
```

Comment:
FYI, you can inline this into the group since you dont reference it anywhere else

Reply:
I had it broken out cos I wanted it to show up in the all actions menu - but even nested it shows up - so I've fixed per your suggestion.